package-lock.json

```diff
@@ -11327,7 +11327,7 @@
       "dependencies": {
         "@mrmlnc/readdir-enhanced": "^2.2.1",
         "@nodelib/fs.stat": "^1.1.2",
-        "glob-parent": "^3.1.0",
+        "glob-parent": "^5.1.2",
```

Comment:
I see that glob-parent have v6 as the latest major, do we want to bump to that? it got some bug fix to resolve a ReDoS vulnerability but I guess Snyk doesn't care about this particular issue

Reply:
Snyk recommended we switch to 5.1.2. Maybe they don't yet know about v6. If it's okay, I'll stick with v5 for now since that's what I vetted through glob-parent's release notes.

Reply:
ah ok, yeah sounds good!